oauth2server/apps/tokens/decorators.py

```diff
@@ -196,6 +197,10 @@ def _validate_grant_type(request):
                 stdlogger.warning( "Raised InvalidUserCredentialsException")
                 raise InvalidUserCredentialsException()
 
+            if not user.account_is_verified:
+                stdlogger.warning("Raised UserAccountNotVerified")
+                raise UserAccountNotVerified
```

Comment:
Should `raise UserAccountNotVerified` have parentheses to be consistent with other raised exceptions here? Still works fine however.

Reply:
Good catch! still works but don't know why... fixed now, and also removed some white space.